portfolio/src/main/webapp/index.html

```diff
@@ -131,33 +131,72 @@ <h5>My Favourite Songs</h5>
-                  gave product designers insight as to where potential clients were having trouble with the trial and steps that may need
-                  improving.</p>
+          <h2 class="display-3 text-center">Experience</h2>
+          <div class="card">
+            <div class="card-header"> 
+                <ul class="nav nav-tabs card-header-tabs"  id="experience-tabs" role="tablist">
+                    <li class="nav-item">
+                        <a class="nav-link active" id="work-tab" data-toggle="tab" href="#work" role="tab" aria-controls="work" aria-selected="true">Work</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" id="projects-tab" data-toggle="tab" href="#projects" role="tab" aria-controls="projects" aria-selected="false">Projects</a>
+                    </li>
+                </ul>
             </div>
-          </div>
-          <div id="projects">
-            <h3>Projects</h3>
-            <div id="mangatracker">
-              <h4>Manga Tracker</h4>
-              <p>As mentioned earlier in my interests, I enjoy reading manga and I used to read them from websites online.
-                I found that checking the website to see if there was a new chapter for all of the manga that I was reading
-                was very tedious and was often a waste of time if there were no new chapters. Their sites were usually filled
-                with lots of ads and popups which made visiting them very unpleasant. There was also pressure to read the 
-                chapters as soon as they were released in order to avoid getting spoiled from other sources such as social media 
-                or friends. I decided to make a web scraper using python that would find out if there was a new release for the 
-                manga that I read and would send an email with the chapters as attachments. I had set this script to run every hour 
-                so I now get chapters almost as soon as they are released and the emails contain just the pages with no ads so
-                it makes for a pleasurable reading experience.</p>
+
+            <div class="card-body">
+                <div class="tab-content" id="experience-content">
+                    <div class="tab-pane fade show active" id="work" role="tabpanel" aria-labelledby="work-tab">
+                      <div id="fivetran">
+                        <h4>Fivetran</h4>
+                        <p>I was a Product Analyst Intern at Fivetran from the beginning of June until the end of July in 2019. One of the
+                            projects I did there was to create a dashboard that illustrated new user retention across trial steps in a given month. This 
+                            gave product designers insight as to where potential clients were having trouble with the trial and steps that may need
+                            improving.</p>
```

Comment:
Idk much about styling - But please verify if the indentation and spacing is as per Google Html/Css guide.

Reply:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

In the first link it says to have at least 4 spaces for line wrapping but I think it refers to the element tags themselves because in the second link there are <p> tags that cross multiple lines but only use two spaces. Should I go ahead and change all my p tags that cross multiple lines to be indented with two spaces?

Reply:
Yes, we can follow that. Just  wanted us to be clear on the guidelines.